src/app/modules/products/components/featured-product-card/featured-product-card.component.html

```diff
@@ -10,7 +10,7 @@ <h2 class="featured-product-card-body-title">{{product.name}}</h2>
         <p class="featured-product-card-body-description" *ngFor="let description of product.displayedDescription">{{description}}</p>
         <div class="featured-product-card-body-price-container">
             <div class="featured-product-card-body-price">{{product.price}}</div>
-            <button class="featured-product-card-body-get-notified" (click)="onClick()">Get notified</button>
+            <button class="featured-product-card-body-get-notified" (click)="showModal()">Get notified</button>
```

Comment:
the method name should always refer to the user action. Here, we should use onGetNotifiedClicked